src/AI/agents/.env.example

```
@@ -8,9 +8,9 @@ AZURE_API_KEY=your_azure_api_key
 # AZURE_OPENAI_ENDPOINT=https://your-resource.openai.azure.com/  # Optional - default set in base_config.py
 
 # =============================================================================
-# OPTIONAL: Gitea proxy (for cloning and pushing branches)
+# REQUIRED: Gitea proxy (for cloning and pushing branches)
 # =============================================================================
-# GITEA_BASE_URL=http://host.docker.internal/repos  # Local: http://studio.localhost/repos
+GITEA_BASE_URL=http://host.docker.internal/repos  # Local: http://studio.localhost/repos
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)